plugins/modules/junos_interface.py

```diff
@@ -255,12 +348,12 @@ def main():
     neighbors_spec = dict(host=dict(), port=dict())
 
     element_spec = dict(
-        name=dict(),
+        name=dict(required=True),
```

Comment:
top-level `name` cannot be required for modules having `aggregate` key

Reply:
that's right!